### PR TITLE
feat: result table header styles

### PR DESCRIPTION
**Pull request by IrakliJani**
Closes: #2911 

### Description:
update styles for results table

CleanShot 2022-08-22 at 21 02 14@2x

https://www.loom.com/share/6b989f79f8094acb982c3ca3f4d791b1

### CODE REVIEW

File: packages/frontend/src/components/LightdashVisualization/index.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { ChartType } from '@lightdash/common';
-import React, { FC } from 'react';
+import { FC } from 'react';
```
Comment (rephus) [inline packages/frontend/src/components/LightdashVisualization/index.tsx:2]: Unecessary  file change

File: packages/frontend/src/hooks/tableVisualization/getPivotDataAndColumns.ts [hunk 2]
```diff
@@ -109,6 +109,7 @@ const getPivotDataAndColumns = ({
             item: itemsMap[pivotDimension],
         },
     });
+
```
Comment (rephus) [inline packages/frontend/src/hooks/tableVisualization/getPivotDataAndColumns.ts:112]: unecessary file change

**Comment by rephus** [comment]
E2E tests are failing 
image
It sounds like it can't find the header now that you have changed the format, so you should update the e2e test to match the new component.

**Review by rephus — CHANGES_REQUESTED** [review]
I can't update the column name labels , I think that's what  `  getHeader,`
   ` getDefaultColumnLabel,` are for
Peek 2022-08-23 12-20

**Comment by IrakliJani** [comment]
@rephus thanks for the review:

- fixed overriding header names
- fixed existing e2e tests
- added e2e to test hiding/showing tables from the config
- added e2e to test overriding column names
- bonus: updated chart type dropdown switcher to use Menu instead of custom button wrapper (see screens)


CleanShot 2022-08-23 at 16 50 28@2x

vs old

CleanShot 2022-08-23 at 16 51 35@2x

**Comment by rephus** [comment]
Everything looks ok to me, 

but this E2E test is failing 
image

Let me know if you need help debugging, you can download the videos or screenshots of cypress on github from here https://github.com/lightdash/lightdash/actions/runs/2911819474

**Comment by IrakliJani** [comment]
@rephus I thought it was a flaky test. then I thought I was going mad. I think what was happening is that you added two extra tables to the jaffle dashboard to test the PR. and e2e also uses jaffle dashboard.

**Comment by rephus** [comment]
> @rephus I thought it was a flaky test. then I thought I was going mad. I think what was happening is that you added two extra tables to the jaffle dashboard to test the PR. and e2e also uses jaffle dashboard.

Damn, I'm so sorry, you were right, I did some manual testing on jaffle dashboard and I broke the tests without noticing >_<

My fault, sorry. 

Everything looks great then. Let's :sheep: it ! Great work.

**Pull request merged**